docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.8"
```

Comment:
I have no idea why, but I've been told that we are not allowed to update docker-compose.yml. Please make sure to revert your changes before you can merge your PR.